recipes/lapack/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# bug 145
+if test `uname` = "Linux"
+then
+  sed -e 's|/CMAKE/|/cmake/|' -i CBLAS/CMakeLists.txt
+fi
+
+# lapack_testing.py does not support py3k, correction sent
+cp ${RECIPE_DIR}/lapack_testing.py .
```

Comment:
> Yes it adds py 2/3 support, but I sent the entire file instead of a real patch as it basically changed the whole file because of the fixed indentation.

👍 Let's leave it alone then.

> I do not have the revision either because they are migrating to git and told me it will be applied once the migration is over.

Oh, so they have already accepted it. That's great! 😄

Honestly, a link to the discussion where this is proposed should be sufficient.